src/CMakeLists.txt

```diff
@@ -154,4 +157,3 @@ add_subdirectory(ShockCloud)
 add_subdirectory(PassiveScalar)
 add_subdirectory(FCQuantities)
 add_subdirectory(SphericalCollapse)
-add_subdirectory(NSCBC)
```

Comment:
Is this change a typo? I think we want to keep this test problem.

Reply:
oops. yeah this is a typo